### DISCUSSION TITLE
Port activities

### DISCUSSION THREAD

**Discussion by hajarpb** [USEPA/supply-chain-factors#15] [Q&A] [answered]
Dear Mo, 
I am using US EPA Supply Chain EFs to estimate GHG emissions from some port activities/services paid by one of our clients. 
I Could not find a good match for these activities in the excel file. Is there any reference document describing the details of each EPA industry or commodity that I can look into? these are some of the activity/services that I could not find in EPA list. 

- Terminal Handling
- Wharfage
- Pilotage
- Line Handling
- Tug Hire Shifting
- Dockage
Thank you.

**Comment by MoLi7** [comment, marked as answer]
Hi @hajarpb 

EPA doesn't provide a curated industry or commodity list. The industry and commodity presented in the Supply Chain EFs spreadsheets are curated by the BEA (Bureau of Economic Analysis) and referred to as BEA sectors. You can find their reference in [a BEA input-output table](https://www.bea.gov/industry/input-output-accounts-data), such as [the Use table](https://apps.bea.gov/industry/xls/io-annual/IOUse_Before_Redefinitions_PRO_DET.xlsx) (click the link to download it). The BEA sectors are further connected to NAICS (North American Industry Classification System) industries (see the 'NAICS Codes' tab in the Use table you just downloaded) that are more detailed and very industry specific.

Now back to your question: you couldn't find anything matching 'port activities/services' in the BEA sectors, then you should search it in NAICS, e.g. Google 'port activities NAICS'. You should see a suggested NAICS code 488310. You can further [look it up in NAICS](https://www.census.gov/naics/?input=488310&year=2012&details=488310) to examine the details. It looks to me that 488310 is the industry that covers the activities/services you are looking for. Next, you should go back to the 'NAICS Codes' tab in the BEA Use table and find which BEA sector corresponds to 488310 or its parent codes including 48831, 4883, and 488. You should find a match for BEA 48A000. Now you can find it in the Supply Chain EFs Detail spreadsheets and use the factors for your analysis.

Hope this helps.

Cheers,
Mo

**Reply by hajarpb** [reply]
Thanks Mo. 
This was really helpful.
